tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -214,52 +213,31 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 		})
 
 		It("verifies that Component 2 can be created and build PipelineRun is created for it in dev namespace and succeeds", func() {
+			GinkgoWriter.Printf("\n\nadditionalComponentDetected :\n %s", additionalComponentDetected)
 			component2, err = fw.AsKubeAdmin.HasController.CreateComponentFromStub(additionalComponentDetected, devNamespace, "", "", applicationNameDefault)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(fw.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component2, "", 2)).To(Succeed())
 		})
 
-		It("verifies that a release PipelineRun for each Component is created in managed namespace.", func() {
-			Eventually(func() bool {
-				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
-				if err != nil || prList == nil || len(prList.Items) < 1 {
-					GinkgoWriter.Printf("Error getting Release PipelineRun:\n %s", err)
-					return false
-				}
-				foudFirstReleasePr := false
-				for _, pr := range prList.Items {
-					if strings.Contains(pr.Name, "release-pipelinerun") {
-						if !foudFirstReleasePr {
-							releasePrName = pr.Name
-							foudFirstReleasePr = true
-						} else {
-							additionalReleasePrName = pr.Name
-						}
-					}
-				}
-
-				return strings.Contains(releasePrName, "release-pipelinerun") &&
-					strings.Contains(additionalReleasePrName, "release-pipelinerun")
-			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
-		})
-
 		It("verifies a release PipelineRun for each component started in managed namespace and succeeded.", func() {
-			Eventually(func() bool {
-
-				releasePr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(releasePrName, managedNamespace)
-				if err != nil {
-					GinkgoWriter.Printf("\nError getting Release PipelineRun %s:\n %s", releasePr, err)
-					return false
-				}
-				additionalReleasePr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(additionalReleasePrName, managedNamespace)
-				if err != nil {
-					GinkgoWriter.Printf("\nError getting PipelineRun %s:\n %s", additionalReleasePr, err)
-					return false
+			prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
```

Comment:
why are we doing this? We can't use any other functions that we already have to search for the proper release pipelinerun?